pkg/apis/core/v1beta1/constants/types_constants.go

```diff
@@ -103,6 +103,12 @@ const (
 	// GardenerOperationReconcile is a constant for the value of the operation annotation describing a reconcile
 	// operation.
 	GardenerOperationReconcile = "reconcile"
+	// GardenerReconciliationRequest is a constant for an annotation on a resource that describes the timestamp when a reconcile has been requested.
+	// It is only used to guarantee an update event for watching clients in case the operation-annotation is already present.
+	GardenerReconciliationRequest = "gardener.cloud/reconciliation-request"
+	// GardenerDeletionRequest is a constant for an annotation on a resource that describes the timestamp when a deletion has been requested.
+	// It is only used to guarantee an update event for watching clients in case the deletion-annotation is already present.
+	GardenerDeletionRequest = "gardener.cloud/deletion-request"
```

Comment:
I don't think we need two annotations, just use `gardener.cloud/timestamp` - it can be updated for both reconciliations and deletions.